### ISSUE TITLE
Large-scale detection and categorization of oil spills from SAR images with deep learning 

### ISSUE THREAD

**Issue by AkiraTOSEI** [AkiraTOSEI/ML_papers#86]
# TL;DR
A study of oil spill detection from synthetic aperture radar (SAR) images from a satellite. We used Unet to perform segmentation, which was comparable to the accuracy of human detection. Using it, they were able to confirm the spill from satellite photos of the period in history when the spill actually occurred. However, the classification of the type did not work, and we presume that mislabeling may be the cause.
![image](https://user-images.githubusercontent.com/24524018/92561033-61f68a80-f2ae-11ea-8b94-a667215f09f9.png)

# Why it matters:

# Paper URL

# Submission Dates(yyyy/mm/dd)
https://arxiv.org/abs/2006.13575

# Authors and institutions

# Methods

# Results

# Comments
